ui/components/src/ui-components/properties/LinkHandler.tsx

```diff
@@ -80,24 +78,24 @@ function renderHighlighted(text: string, highlight: (text: string) => React.Reac
 }
 
 /** Returns true if property record has an anchor tag
- * @public
+ * @deprecated

```

Comment:
When adding `@deprecated` tag, please also specify what should consumers do if they're using the API

Reply:
BTW I think the `@public` should stay as-is and `@deprecated` should be added **in addition** to it.